CONTRIBUTING.md

```diff
@@ -1,27 +1,167 @@
-Before contributing you should sign [Canonical's contributor agreement][1],
-it’s the easiest way for you to give us permission to use your contributions.
+# Contributing to snapd
 
-## Pull Requests and tests
+We are an open source project and welcome community contributions, suggestions,
+fixes and constructive feedback.
 
-We need to verify that the code functionality and quality is not degraded
-by additions before merging any changes to snapd's codebase. For each PR
-we run checks in three different groups: static, unit and spread.
+If you'd like to contribute, you will first need to sign the Canonical
+contributor agreement. This is the easiest way for you to give us permission to
+use your contributions. In effect, you’re giving us a licence, but you still
+own the copyright — so you retain the right to modify your code and use it in
+other projects.
 
-Static test use several code analysis tools present in the GoLang ecosystem
+The agreement can be found, and signed, here:
+https://ubuntu.com/legal/contributors
+
+If you have any questions, please reach out to us on our forum:
+https://forum.snapcraft.io/c/snapd/5
+
+## Contributor guidelines
+
+Contributors can help us by observing the following guidelines:
+
+- Commit messages should be well structured.
+- Commit emails should not include non-ASCII characters.
+- PR titles should start with a prefix denoting the scope affected by the changes
+(e.g. `<feature name>`,`<package name>`) followed by a semicolon. 
```

Comment:
I actually don't think we use <feature name>:  , we have more extensive info on this in CODING, not sure how you want to approach tweaking this

Reply:
Thanks! I've removed this and instead added a link to CODING.md beneath this list.